### PR TITLE
add amp-script sample

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

File: examples/source/1.components/amp-script/index.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    button.addEventListener('click', () => {
+      const h1 = document.createElement('h1');
+      h1.textContent = 'Hello World 2!';
+      document.body.appendChild(h1);
+    });
+  </script>
+  <!--
+      ## Using the fetch API
+
+      `amp-script` supports using the fetch API. To be able to update the page on load, we need to use the `fixed-height` layout instead of the `container` layout.
```
Comment (fstanis) [inline examples/source/1.components/amp-script/index.html]: "...and the height has to be 300px or less"

File: examples/source/1.components/amp-script/index.html [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+      -->
+      <amp-script layout="container" src="<% hosts.platform %>/hello-world.js" class="amp-script-sample" >
+        <button>Click!</button>
+      </amp-script>
+
+      <!-- 
+        This is the script in [hello-world.js](hello-world.js):
+
+        ```js
+const button = document.querySelector('button');
```
Comment (fstanis) [inline examples/source/1.components/amp-script/index.html]: I'd give this button an id, it's very unusual to attach a listener to a tag selector, so I fear that might confuse people.

File: examples/source/1.components/amp-script/index.html (hunk 1 above)
Comment (sebastianbenz) [reply examples/source/1.components/amp-script/index.html]: done

File: examples/source/1.components/amp-script/index.html (hunk 2 above)
Comment (sebastianbenz) [reply examples/source/1.components/amp-script/index.html]: done (that's how we do it in the amp script doc)

**Comment by sebastianbenz** [comment]
Adressed the comments. PTAL

File: examples/source/1.components/amp-script/index.html [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+      <script id="hello-world" type="text/plain" target="amp-script">
+    const button = document.querySelector('#hello2');
+    button.addEventListener('click', () => {
+      const h1 = document.createElement('h1');
+      h1.textContent = 'Hello World 2!';
+      document.body.appendChild(h1);
+    });
+  </script>
+
+  <!-- Note: Good to know: calling `document.body.appendChild(...)` will add an element to the `amp-script` element and not to the actual `body`. -->
```
Comment (fstanis) [inline examples/source/1.components/amp-script/index.html]: This makes it sound like `appendChild` is special, not `document.body`.

nit: I know `Note:` has special meaning and creates that callout box, but it's still weird to have two colons this close in a sentence. I'd remove "Good to know:".

Comment (sebastianbenz) [reply examples/source/1.components/amp-script/index.html]: done

**Comment by sebastianbenz** [comment]
PTAL

**Comment by jaygray0919** [comment]
Examples would be helpful. We have struggled with techniques to 'roll-up' multiple `file.js` into a single `main.js`, with `amp-script`-eligible parts of the `DOM`, etc. Our main goal is to implement our `d3.js` applications using `amp-script` and have not yet succeeded.

**Comment by sebastianbenz** [comment]
@jaygray0919 d3 is tricky, I think @alabiaga looked in this.

**Comment by sebastianbenz** [comment]
Thanks for the review @fstanis!

**Pull request merged**